src/RegisterQD.jl

```diff
@@ -17,13 +17,15 @@ include("translations.jl")
 include("rigid.jl")
 include("affine.jl")
 include("gridsearch.jl")
+include("SD_calculator.jl")
```

Comment:
I think it's better in utils too. I only had it in a separate file to avoid getting it steamrolled by all of the other changes over the last couple of days.